encoding/protobuf/protoc-gen-yarpc-go/internal/lib/lib.go

```diff
@@ -30,12 +30,12 @@ import (
 	"go.uber.org/yarpc/internal/protoplugin"
 )
 
-const tmpl = `{{$packagePath := .GoPackage.Path}}
+const tmpl = `{{$packagePath := .GoPackage.Path}}{{$packageName := .GoPackage.Name}}
```

Comment:
FYI, if you're putting these on the same line just to have the "Code generated
by" at the top of the file, you can tell text/template to ignore leading/
trailing whitespace by using `{{-` and/or `-}}`.

```
const tmpl = `
{{- $packagePath := .GoPackage.Path -}}
{{- $packageName := .GoPackage.Name -}}
// Code generated by [..]
```

Reply:
Thanks for pro tip! If it's not a big deal, I'm just going to keep it as-is as this is how the rest of the template operates.

Reply:
Yeah, this was just an FYI.